library/neutron_network.py

```diff
@@ -18,6 +18,8 @@
 
 try:
     from neutronclient.neutron import client
+    from keystoneauth1.identity import v2
```

Comment:
Actually it provides both v2 and v3 stuffs, https://github.com/openstack/keystoneauth  But I'm calling it "v2" because that's what other modules called it.